streamz/sinks.py

```diff
@@ -0,0 +1,96 @@
+from tornado import gen
+
+from streamz import Stream
+
+_global_sinks = set()
+
+
+class Sink(Stream):
+
+    _graphviz_shape = 'trapezium'
+
+    def __init__(self, upstream, **kwargs):
```

Comment:
Does this need special removal methods?

Reply:
Answered (with a question, really) here: https://github.com/python-streamz/streamz/pull/386#discussion_r523273681